[2023-10-13 21:04:03,509] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: consultar_info_covid_api.load_data_redshift scheduled__2023-10-12T00:00:00+00:00 [queued]>
[2023-10-13 21:04:03,515] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: consultar_info_covid_api.load_data_redshift scheduled__2023-10-12T00:00:00+00:00 [queued]>
[2023-10-13 21:04:03,515] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-13 21:04:03,515] {taskinstance.py:1377} INFO - Starting attempt 15 of 20
[2023-10-13 21:04:03,516] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-13 21:04:03,527] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): load_data_redshift> on 2023-10-12 00:00:00+00:00
[2023-10-13 21:04:03,530] {standard_task_runner.py:52} INFO - Started process 194 to run task
[2023-10-13 21:04:03,532] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'consultar_info_covid_api', 'load_data_redshift', 'scheduled__2023-10-12T00:00:00+00:00', '--job-id', '407', '--raw', '--subdir', 'DAGS_FOLDER/api_covid.py', '--cfg-path', '/tmp/tmp4otvvqpf', '--error-file', '/tmp/tmpu7f0dhcm']
[2023-10-13 21:04:03,532] {standard_task_runner.py:80} INFO - Job 407: Subtask load_data_redshift
[2023-10-13 21:04:03,571] {task_command.py:371} INFO - Running <TaskInstance: consultar_info_covid_api.load_data_redshift scheduled__2023-10-12T00:00:00+00:00 [running]> on host 738c88afe20a
[2023-10-13 21:04:03,617] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Axel Villegas Luna
AIRFLOW_CTX_DAG_ID=consultar_info_covid_api
AIRFLOW_CTX_TASK_ID=load_data_redshift
AIRFLOW_CTX_EXECUTION_DATE=2023-10-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=15
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-12T00:00:00+00:00
[2023-10-13 21:04:31,072] {logging_mixin.py:115} INFO - Exito
[2023-10-13 21:04:31,072] {python.py:173} INFO - Done. Returned value was: None
[2023-10-13 21:04:31,082] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=consultar_info_covid_api, task_id=load_data_redshift, execution_date=20231012T000000, start_date=20231013T210403, end_date=20231013T210431
[2023-10-13 21:04:31,106] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-13 21:04:31,125] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
